local.c

```diff
@@ -835,6 +835,23 @@ static int add_attr_to_channel(struct iio_channel *chn,
 	union iio_pointer p = { .chn = chn, };
 	struct iio_attr_list *attrs;
 	int ret;
+	const char *lbl_suffix = "_label";
+	size_t lbl_len = strlen(lbl_suffix);
+	const char *chn_label_ptr = NULL;
+	char label[512];
+	const char *dev_id;
+
+	if (strlen(name) >= lbl_len &&
+		!strcmp(name + strlen(name) - lbl_len, lbl_suffix)) {
```

Comment:
This is more subjective - Since `lbl_suffix` is only used once - why not just use the string constant `"_label"` in the code? In theory - it should be done by the cpp - but this will ensure that. Same for the use of `strlen(lbl_suffix)` just use `strlen("_label")` in two places. Makes it easier to read IMHO.

Reply:
I created `lbl_suffix` because `"_label"` is used in 2 places and I believed it would be easier to change that string only once if needed. But now that I think better, it's unlikely that "_label" will change in the future. I agree that your suggestion is just easier to read.

Reply:
>  But now that I think better, it's unlikely that "_label" will change in the future

That would be an ABI breakage that won't happen in the kernel :)